Isodose/Logic/vtkMRMLIsodoseNode.h

```diff
@@ -94,12 +95,27 @@ class VTK_SLICER_ISODOSE_LOGIC_EXPORT vtkMRMLIsodoseNode : public vtkMRMLNode
   vtkSetMacro(ShowDoseVolumesOnly, bool);
   vtkBooleanMacro(ShowDoseVolumesOnly, bool);
 
+  /// Get/Set reference dose value
+  vtkGetMacro(ReferenceDoseValue, double);
+  vtkSetMacro(ReferenceDoseValue, double);
+
+  /// Get/Set dose units type
+  vtkGetMacro(DoseUnits, DoseUnitsType);
+  vtkSetMacro(DoseUnits, DoseUnitsType);
+
+  /// Get/Set relative representation flag
+  vtkGetMacro(RelativeRepresentationFlag, bool);
+  vtkSetMacro(RelativeRepresentationFlag, bool);
+  vtkBooleanMacro(RelativeRepresentationFlag, bool);
+
 protected:
   vtkMRMLIsodoseNode();
   ~vtkMRMLIsodoseNode();
   vtkMRMLIsodoseNode(const vtkMRMLIsodoseNode&);
   void operator=(const vtkMRMLIsodoseNode&);
 
+  void SetDoseUnits(int doseUnits = 0);

```

Comment:
It's strange that there is a default value for the argument in a Set function. Why?

Reply:
There is no default value anymore.